bin/bundler

```diff
@@ -6,11 +6,11 @@
 # this file is here to facilitate running it.
 #
 
-require 'pathname'
-ENV['BUNDLE_GEMFILE'] ||= File.expand_path("../../Gemfile",
-  Pathname.new(__FILE__).realpath)
+require "pathname"
+ENV["BUNDLE_GEMFILE"] ||= File.expand_path("../../Gemfile",
+                                           Pathname.new(__FILE__).realpath)
```

Comment:
This isn't actually how we do this.

```rb
ENV["BUNDLE_GEMFILE"] ||= File.expand_path(
  "../../Gemfile",
  Pathname.new(__FILE__).realpath
)
```

Is something off in our rubcop config, that it's allowed this?